src/sdbus/autodoc.py

```diff
@@ -126,3 +126,7 @@ def setup(app: Sphinx) -> None:
     app.add_autodocumenter(DbusMethodDocumenter)
     app.add_autodocumenter(DbusPropertyDocumenter)
     app.add_autodocumenter(DbusSignalDocumenter)
+    return {
```

Comment:
`src/sdbus/autodoc.py:129: error: No return value expected`

The function annotation needs also to be changed.

Reply:
I need to setup mypy CI